src/azul/indexer/__init__.py

```diff
@@ -255,6 +280,30 @@ def __str__(self) -> str:
         """
         return f'{self.name}:{self.prefix}'
 
+    def contains(self, other: 'SimpleSourceSpec') -> bool:
+        """
+        >>> p = SimpleSourceSpec.parse
+
+        >>> p('https://foo.edu:2').contains(p('https://foo.edu:2'))
+        True
+
+        >>> p('https://foo.edu:2').contains(p('https://foo.edu:2/1'))
+        True
+
+        >>> p('https://foo.edu:2').contains(p('https://foo.edu:'))
+        False
+
+        >>> p('https://foo.edu:2/1').contains(p('https://foo.edu:'))
+        False
+
+        >>> p('foo:').contains(p('foo:'))
+        True
+        """
```

Comment:
No need to cover `super().contains(other)` completely again. Technically, you'd have to cover every possible combination of the terms in this method. So, `super().contains(other) and and self.name == other.name`, `not super().contains(other) and and self.name == other.name`, `super().contains(other) and and self.name != other.name` and `not super().contains(other) and and self.name != other.name`.

Then again this method is trivial, so just remove the doctests here.